monolithic/pkg/assembler.go

```diff
@@ -101,6 +101,20 @@ func RunMonolithicLamassuPKI(conf MonolithicConfig) (int, int, error) {
 				Port:               0,
 				Protocol:           cconfig.HTTP,
 			},
+			FilesystemStorage: cconfig.FSStorageConfig{
+				ID:   "fs",
+				Type: cconfig.LocalFilesystem,
+				Config: map[string]interface{}{
+					"storage_directory": "/home/ubuntu/dev/lamassuiot/gocloud",
```

Comment:
This should be a configuration value or use the same approach that the FS Cryptoengine